packages/ui/client/composables/client/index.ts

```diff
@@ -75,6 +77,7 @@ watch(
         client.rpc.getConfig(),
       ])
       client.state.collectFiles(files)
+      unhandledErrors.value = await client.rpc.getUnhandledErrors()
```

Comment:
Should we put it inside the `Promise.all` a few lines earlier?

Reply:
ohh yeah, will do

Reply:
done